src/PhotoPage.tsx

```diff
@@ -35,6 +35,13 @@ export const PhotoPage = () => {
         url="/"
         text={"go back"}
       />
+      <Typography sx={{ paddingTop: 4 }} variant="h3" color={MAIN_COLOR}>
+        SOLEDAD MILLION GALLERY
+      </Typography>
+      <Typography fontSize="15px" color={THIRD_COLOR}> 
+        Want to thank Henry and James? Leave a tip 
+        <a href="buymeacoffee.com/jamesphotographyco" target="_blank" rel="noopener noreferrer" style={{ color: MAIN_COLOR, textDecoration: 'none' }}> here!</a>
```

Comment:
fantastic integration here! i love it

Reply:
Do you have another method of adding links? or is using an attribute href the correct method in typescript?